.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore common IDE files
+/.idea
```

Comment:
Just let me figure out how on earth to accept this change request. Do I just create a new PR? The github UI is like a labyrinth!